src/build/webpack.config.js

```diff
@@ -170,7 +170,9 @@ function composeMaterialVersion(material) {
   let commitHash = "";
   try {
     commitHash = childProcess
-      .execSync("git rev-parse --short HEAD", { cwd: material })
+      .execSync("git -c safe.directory=* rev-parse --short HEAD", {
```

Comment:
I think a comment could be useful here
```suggestion
      // see https://github.com/Zenika/sensei/pull/154 for the why of safe.directory here
      .execSync("git -c safe.directory=* rev-parse --short HEAD", {
```